docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -530,7 +530,7 @@ Packaging Ansible or wanting to build a local package yourself, but don't want t
 
 .. note::
 
-	If you are creating your own Ansible package, you must also download or package ``ansible-base`` as part of your Ansible package. You can download it as https://pypi.python.org/packages/source/a/ansible-base/ansible-base-{{VERSION}}.tar.gz.
+	If you are creating your own Ansible package, you must also download or package ``ansible-core`` as part of your Ansible package. You can download it as https://pypi.python.org/packages/source/a/ansible-core/ansible-core-{{VERSION}}.tar.gz.
 
```

Comment:
I can't get this url to work in any variation.

Reply:
ansible-core is only for 2.11+

We have a beta2 up that seems to work:

https://pypi.python.org/packages/source/a/ansible-core/ansible-core-2.11.0b2.tar.gz

Reply:
Thanks @samcann @sivel. I'll try adding a list here to show the different versions and the URLs that apply to them.

Reply:
@acozine Reviewed the PR and looks like you caught the mistakes. LGTM.

Reply:
Addressed in 578a6191ef0bb7a2fa31e0f2dc5e40065afc67ed.